backend/Dockerfile.cloud

```diff
@@ -0,0 +1,109 @@
+FROM python:3.11.7-slim-bookworm
+
+LABEL com.danswer.maintainer="founders@danswer.ai"
+LABEL com.danswer.description="This image is the web/frontend container of Danswer which \
```

Comment:
A lot of the comments and things are not correct. I would say this just should not be used by the community.

Reply:
let's remove this one